presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -51,6 +77,34 @@ public static double abs(@SqlType(StandardTypes.DOUBLE) double num)
         return Math.abs(num);
     }
 
+    private static SqlScalarFunction decimalAbsFunction()
```

Comment:
You do not have to build this manualy.
Use annotation based definition. See `DecimalOperators.add` for example